modules/replicator/src/main/java/org/apache/ignite/internal/replicator/ReplicaManager.java

```diff
@@ -1028,4 +1087,357 @@ public Set<ReplicationGroupId> startedGroups() {
                 .map(Entry::getKey)
                 .collect(toSet());
     }
+
+    @TestOnly
+    public boolean isReplicaPrimaryOnly(ReplicationGroupId groupId) {
```

Comment:
Do we really need it? Why not to user. PD.getPrimary() check whether replica in assignments or not?

Reply:
seems it's different, it tells nothing about replica state in terms of lifecycle